samples/ResourceGroup/copy-resourceGroup-tag/README.md

```diff
@@ -0,0 +1,34 @@
+# Copy resource group tag to resource
+
+Copy a named tag from the resource group tag to the resource.  For example, a tag named example on the resource group will be copied to the resources.  Change the word example in the policy rule to change the name of the tag copied to a resource.
```

Comment:
Use _italics_ or the  `code` [markdown ](https://guides.github.com/features/mastering-markdown/) around `example` to  make it stand out